docs/tutorial.md

```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
 4. Select __OK__ to close the window.  
-5. Select __Bun Color__ to make it an active layer. 
+5. Select __Bun Color__ layer to make it an active layer. 
 6. Go to __File > Open...__ and open the saved burger image reference.
-7. Select __Color Picker__ to pick the bun color from the reference image and switch back to the burger project under the __Image Window__. You can create and modify your own color by double-clicking on the primary or secondary color boxes under [__Palette__](palette.md).
+7. [Select __Color Picker__ tool](coloring.md#select-an-exact-color-match-from-an-image) to pick the bun color from the reference image and switch back to the burger project under the __Image Window__. You can create and modify your own color by double-clicking on the primary or secondary color boxes under [__Palette__](palette.md).
 8. Select __Paint Brush__ to color in the entire burger. You can enlarge the size of the __Paint Brush__ in the __Tools Settings Bar__ to speed up the process.
-9. Select __Eraser__ to clean up any color that doesn't belong. 
-10. Save your progress and it should look like this: 
+9. Select the [__Eraser__ tool](remove.md#eraser) to clean up any color that doesn't belong. 
+10. Save your progress and it should look something like this: 
 ![color bun](img/burger/color_bun.gif)
 
 ### __Cheese, Tomatoes & Lettuce__ ###
 We'll now work on the cheese, tomatoes, and lettuce:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Bun Color__ layers.
-2. Open __Layer Properties__ for the new layer and enter __Other Color__ in the text box. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Other Color__. 
 3. Select __OK__ to close the window. 
-4. Select __Other Color__ to make it an active layer. 
+4. Select __Other Color__ layer to make it an active layer. 
 5. Using the same concept as the bun, select __Color Picker__ to pick the colors from the reference image to apply to the project for the cheese, tomatoes, and lettuce. 
-6. Select __Paint Brush__ and __Eraser__ to color in the appropriate areas. 
+6. Select the __Paint Brush__ and __Eraser__ tools to color in the appropriate areas. 
 7. Save your progress and it should look like this: 
 ![other color](img/burger/other_color.png)
 
 ### __Patty__ ###
 For the patty, we want to create a gradient to replicate the grilled side of the patty:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Other Color__ layers.
-2. Open __Layer Properties__ for the new layer, enter __Patty Color__, and enter opacity to 90. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Patty Color__ and enter an **Opacity** value of 90. 
 3. Select __OK__ to close the window. 
-4. Select __Patty Color__ to make it an active layer. 
+4. Select __Patty Color__ layer to make it an active layer. 

```

Comment:
keep original

Reply:
This could be confusing for someone as they could click on the Patty!! not the Patty Color Layer... wonder if you are too close to creating the instructions to notice this...but it was confusing when I read it.

Reply:
but that's because i put the note at the top to read the setup page first to understand. this is also a perspective of when some doesnt understand the concept, we have mentioned to check out the layer concept. from there, they should be able to see what an active layer mean and implies. let me know your thoughts. i;m also not too against putting the word there if you feel its necessary. i just thought its too wordy because i thought its obvious from doing the setup page already.